### PR TITLE
Pass images

### PR DESCRIPTION
**Pull request by clyang82**
fixed: https://github.com/open-cluster-management/backlog/issues/14157

Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -159,21 +159,21 @@ e2e-tests: test-e2e-setup
...
 test-e2e-setup:
 	@echo "Seting up E2E Tests environment..."
-ifdef COMPONENT_IMAGE_NAME
+ifdef COMPONENT_IMAGE_PIPELINE
 	# override the image for the e2e test
-	@./cicd-scripts/setup-e2e-tests.sh -a install -i $(COMPONENT_IMAGE_NAME)
+	@./cicd-scripts/setup-e2e-tests.sh -a install -p $(COMPONENT_IMAGE_PIPELINE)
+endif
+ifdef COMPONENT_IMAGE_NAMES
+	# override the image for the e2e test
```
Comment (morvencao) [inline Makefile:167]: do we still need this? `COMPONENT_IMAGE_PIPELINE` can be used for local test?

**Review by morvencao — COMMENTED** [review]
can you

File: Makefile (hunk 1 above)
Comment (morvencao) [reply Makefile:167]: forget that, I got that, the pipeline images have different format.

**Review by morvencao — COMMENTED** [review]
/lgtm

File: Makefile (hunk 1 above)
Comment (clyang82) [reply Makefile:167]: `COMPONENT_IMAGE_PIPELINE` is something like `registry.build01.ci.openshift.org/ci-op-1ts27vc8/pipeline` so just use component name as image tag. that is why leave `COMPONENT_IMAGE_NAMES` for local test.

**Pull request merged**